### PR TITLE
feat: all-time ratio card

### PR DESCRIPTION
**Pull request by rayun56**
# feat: all time ratio card

Add an all-time ratio card below the all-time statistics on the sidebar (see below)
As there is no readily available session ratio stat, only included this extra card for all time stats
image

# PR Checklist

- [x] I've started from master
- [x] I've only committed changes related to this PR
- [x] All Unit tests pass
- [x] I've removed all commented code
- [x] I've removed all unneeded console.log statements

### CODE REVIEW

File: src/components/Navbar/AllTimeStats.vue
Comment (Larsluph) [inline src/components/Navbar/AllTimeStats.vue:1]: Why create a new component when 90% is duplicated code? I suggest to incorporate your changes into the existing `TransferStats`

File: src/components/Navbar/TransferStats.vue
Comment (Larsluph) [inline src/components/Navbar/TransferStats.vue:1]: Add changes from AllTimeStats.vue here with `v-if="!isSession"` to prevent duplication

**Comment by rayun56** [comment]
js/Vue is not a language I'm very familliar with, thanks for the suggestions

**Comment by lyaschuchenko** [comment]
Have a bug, its very annoying 
Screenshot_2023-04-27-08-18-16-612_com android chrome-edit

**Comment by Larsluph** [comment]
> Have a bug, its very annoying 

Can you create a new issue for that ?

**Pull request merged**